shared/message/behaviour/MotionCommand.proto

```diff
@@ -27,21 +27,21 @@ import "Vector.proto";
  */
 message MotionCommand {
     enum Type {
-        /// Stop moving and just stand still, (equivalent to a VelocityCommand of {0} m/s)
-        StandStill = 0;
+        /// Stop moving and just stand still (equivalent to disabling the walk engine)
```

Comment:
I dont think this is true. The walk engine could still be enabled (and active) but the robot can be standing still. A veloctiy command of 0m/s and 0rad/s is more correct

Reply:
I remember looking into this because of this comment, and the walk engine disables the main reactor in this case. It doesn't get a 0 walk command.

Reply:
Hmmmm or maybe it does?

Reply:
I'm wrong, Alex is right, it's just a 0 walk command. I'm not sure what I'm 'remembering' then 😆

Reply:
I think, to a certain degree, we should avoid comments in the messages that are dependent on the current implementation in the modules. Just because the current walk engine disables itself for `STAND_STILL` (assume it does for the sake of argument), doesn't mean another walk engine we implement will do the same thing, but if this comment says that it does then it will be confusing later on. In this case, I think the correct comment should be `Stop moving and just stand still`, leave it up to the walk engine how it wants to implement that behaviour

Reply:
Great point. I agree completely. I'll update the comment as such.